CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+
+### Added
+- `tf-modules/cumulus` module now supports a `cmr_custom_host` variable that can be used to set to an arbitrary host for making CMR requests (e.g. `https://custom-cmr-host.com`).
+- Added `buckets` variable to `tf-modules/archive`
+### Changed
```

Comment:
Nit: please include newlines between the headers

Reply:
[`a988867` (#2121)](https://github.com/nasa/cumulus/pull/2121/commits/a988867e4274bc68de14a1c2cc265f3f79bdeac6)